src/HomeWorkApp.java

```diff
@@ -0,0 +1,40 @@
+public class HomeWorkApp {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign(1,2);
+        printColor(101);
+        compareNumbers(2, 1);
+    }
+
+    public static void printThreeWords() {
+        System.out.println("_Orange");
+        System.out.println("_Banana");
+        System.out.println("_Apple");
+    }
+
+    public static void checkSumSign(int a, int b) {
+        if (a + b >= 0) {
+            System.out.println("Сумма положительная");
+        } else {
+            System.out.println("Сумма отрицательная");
+        }
+    }
+
+    public static void printColor(int value) {
+        if (value <= 0){
```

Comment:
```suggestion
        if (value <= 0) {
```
и в других местах также скобку лучше отделать пробелом.
Можете использовать ctrl+alt+L для автоформатирования